package.json

```diff
@@ -81,7 +81,7 @@
   "private": true,
   "scripts": {
     "css": "sass --load-path=node_modules --update app/sass:app/dist --style compressed",
-    "format": "eslint '**/*.js' --fix --ignore-pattern '/app/dist/'",
+    "format": "eslint \"**/*.js\" --fix --ignore-pattern '/app/dist/'",
```

Comment:
Does the second command in this line need to escaped too?

Reply:
Not needed, runs just fine after escaping only the eslint command. :)